ego: #namespace

    #environment variables
    reward_height_b: 0.25
    reward_height_k: 1
    effort_penalty: 0.05   
    effort_max: 11     
    epoch_steps: 256 #length of the simulation
    running_step: 0.01 # amount of time the control will be executed
    
    #qlearn parameters
    alpha: 0.1
    gamma: 0.7
    epsilon: 0.9
    epsilon_discount: 0.999
    nepisodes: 500
    nsteps: 1000
    #running_step: 0.06 # Time for each step
# ego: #namespace

#     #qlearn parameters
    
#     alpha: 0.01 # Learning Rate
#     alpha_decay: 0.01
#     gamma: 1.0 # future rewards value 0 none 1 a lot
#     epsilon: 1.0 # exploration, 0 none 1 a lot
#     epsilon_decay: 0.995 # how we reduse the exploration
#     epsilon_min: 0.01 # minimum value that epsilon can have
#     batch_size: 64 # maximum size of the batches sampled from memory
#     episodes_training: 1000
#     n_win_ticks: 250 # If the mean of rewards is bigger than this and have passed min_episodes, the task is considered finished
#     min_episodes: 100
#     #max_env_steps: None
#     monitor: True
#     quiet: False

#     # Ego environment variables
#     control_type: "effort"
#     min_pole_angle: -0.7 #-23°
#     max_pole_angle: 0.7 #23°
#     max_base_velocity: 50
#     max_base_pose_x: 0.5
#     min_base_pose_x: -0.5
    
#     n_observations: 4 # Number of lasers to consider in the observations
#     n_actions: 2 # Number of actions used by algorithm and task
    
#     # those parameters are very important. They are affecting the learning experience
#     # They indicate how fast the control can be
#     # If the running step is too large, then there will be a long time between 2 ctrl commans
#     # If the pos_step is too large, then the changes in position will be very abrupt
#     running_step: 0.04 # amount of time the control will be executed
#     pos_step: 1.0     # increment in position/velocity/effort, depends on the control for each command
#     init_pos: 0.0 # Position in which the base will start
#     wait_time: 0.1 # Time to wait in the reset phases